src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -228,7 +228,7 @@ private InputMapping exportInput(PKSimProject project, QualifcationConfiguration
          var fileFullPath = Path.Combine(targetFolder, $"{buildingBlockName}{CoreConstants.Filter.MARKDOWN_EXTENSION}");
 
          // Use wait for now until we can support // run of input
-         _markdownReporterTask.ExportToMarkdown(buildingBlock, fileFullPath).Wait();
+         _markdownReporterTask.ExportToMarkdown(buildingBlock, fileFullPath, input.SectionLevel).Wait();
```

Comment:
I have added the section level here which now comes from the QualificationPlan
For example here https://github.com/Open-Systems-Pharmacology/Raltegravir-Model/blob/master/evaluation/Input/Evaluation_plan.json#L444, the level would be 3 which was the default value that was supposed to be used.
If the element is defined somewhere else, its level will be automatically adjusted